File: Programmers/LV0/Python/점의 위치 구하기.py
def solution(dot):
    if dot[0] > 0 and dot[1] > 0:
        answer = 1
    elif dot[0] > 0 and dot[1] < 0:
        answer = 4
    elif dot[0] < 0 and dot[1] > 0:
        answer = 2
    else:
        answer = 3
    return answer


dot = [-3, -2]
print(solution(dot))

File: Programmers/LV0/Go/src/점의 위치 구하기.go
package main

import "fmt"

func solution_120841(dot []int) int {
	if dot[0] > 0 && dot[1] > 0 {
		return 1
	} else if dot[0] < 0 && dot[1] > 0 {
		return 2
	} else if dot[0] < 0 && dot[1] < 0 {
		return 3
	} else {
		return 4
	}
}

func main() {
	dot := []int{2, 4}

	fmt.Println(solution_120841(dot))
}
